conformance/lang/expressions/multiplicative-expr/multiplicative_expr.balt

```diff
@@ -17,7 +14,7 @@ Labels: multiplicative-expr, int, float, decimal
 
 function errorFunction(int a, float b, decimal c) {
     int _ = a * c; // @error * is not allowed with operands of different basic types
-    decimal _ = c / a; // @error / is not allowed with operands of different basic types
```

Comment:
Should we add some set of `output` tests as well to cover allowed combinations

Reply:
Yes, already created https://github.com/ballerina-platform/ballerina-lang/issues/35826. This PR is to just get the existing tests to pass.